docs/dev/features/netapp-mounting/overview.md

```diff
@@ -30,42 +32,38 @@ This controller is built off of the existing `aaw-kubeflow-profiles-controller`
-- The name of the filer that the bucket or share is on. This will be used to populate the secret, as the mounting solution requires the S3 url. 
+This controller is responsible for managing what buckets that a user has access to, as well as creating the secrets associated with a user and all that comes with it. The bucket that is created uses a _hashed_ version of the inputted request path in order to comply with naming conventions and to avoid collision. This controller uses a configmap that the user generates from the UI that requests certain paths in an SVM (in this case an SVM is equivalent to a filer ex fld9filer). Using that configmap, API calls are sent off to the following; current Kubernetes cluster, Microsoft Graph, Ontap.
 
-This works like all the other profiles controllers, and steps can be taken from above as well. The controller watches profile resources and will generate a unique to bucket secret (for as many filers as a user has acccess to there should be a secret) following the naming convention of `*filer-conn-secret` for the profile that contains the following fields
-- S3_ACCESS
-- S3_BUCKET
-- S3_SECRET
-- S3_URL
+The current kubernetes cluster is used for the following
+- Determine whether or not a user secret exists for an svm, and submit a create if needed.
+- Create and modify the end `existing-shares` configmap, which is used by the filer-sidecar-injector.
+- Creating and modifying the `shares-errors` configmap, which shows errors to the user.
+- Grabbing data from the `filers-list` configmap and the secrets needed to interact with the Microsoft Graph Api and Ontap.
 
-[This is the API we are trying to call, but do not have permissions to](https://docs.netapp.com/us-en/ontap-restapi/ontap/protocols_s3_services_svm.uuid_users_endpoint_overview.html#creating-an-s3-user-configuration)
+The Microsoft Graph API is used to get the onpremises name, as the username that the user has in the Netapp system matters as that is how the mapping of permissions is done. 
 
-**Again, this is a work in progress.**
-It will need to change based on work done in the Manage Filers page in Central dashboard when users select what filer (and level of) that they have access to.
-Additionally the contents of the secret will also need additions, as when I was creating this controller I was not aware of the `BUCKET` itself needing to be a part of the secret (you could retrieve it from what populates the Manage Filers page but that is more complicated and another hoop to jump through).
+The Ontap API is the main driver here we query it to;
+- Determine if a user exists S3 side, if not we will create it.
+- Check if a user group exists, if yes add the current user, if not create it and add the user. 
+- Retrieve the actual `nas_path`. This is because the user inputted path will be different from what is actually on the Netapp system and we need that `path` else our request to create the bucket will error out.

```

Comment:
Maybe mention we get that from shares? Just to also introduce that concept in the docs